nats-exporter/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN git clone --branch v0.9.1 https://github.com/nats-io/prometheus-nats-exporte
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -v -a -tags netgo -installsuffix netgo -ldflags "-s -w"
 
 # Final docker image building stage
-FROM alpine:3.15
+FROM quay.io/astronomer/ap-base:3.15.1
```

Comment:
Why switch from alpine to our ap-base image?

Reply:
our ap-base is built on top of alpine - this way we can have common images , thats the core idea. I have tagged @danielhoherd to see if he is okay

Reply:
Using our vendored images is always going to be better for us since we have control over them.